aikau/src/test/resources/testApp/WEB-INF/classes/alfresco/site-webscripts/alfresco/renderers/CommentsList.get.js

```diff
@@ -19,6 +19,7 @@ model.jsonModel = {
          name: "alfresco/renderers/CommentsList",
          id: "COMMENT_LIST",
          config: {
+            addCommentsFullScreen: true,
```

Comment:
I didn't mean that this should be the default... this should be something that the Integrations team can configure when they detect that they're viewing the Document Library in an iframe (so that space is restriced)